tests/backend/mongodb/test_connection.py

```diff
@@ -32,15 +32,15 @@ def mongodb_connection():
                        port=bigchaindb.config['database']['port'])
 
 
-def test_get_connection_returns_the_correct_instance():
+def test_get_connection_returns_the_correct_instance(db_host, db_port):
     from bigchaindb.backend import connect
     from bigchaindb.backend.connection import Connection
     from bigchaindb.backend.mongodb.connection import MongoDBConnection
 
     config = {
         'backend': 'mongodb',
-        'host': 'localhost',
-        'port': 27017,
+        'host': db_host,
+        'port': db_port,
```

Comment:
`db_host` and `db_port` are fixtures, and these fixtures get the their values from the config which in turn is set in some parts via environment variables.

In this case `db_host` is obtained via the environment variable `BIGCHAINDB_DATABASE_HOST`, whereas the `db_port` is obtained via the environment variable `BIGCHAINDB_DATABASE_PORT`.

For docker-based tests referred to in this PR, the environment variables are set in the `docker-compose.yml` file: https://github.com/bigchaindb/bigchaindb/blob/master/docker-compose.yml#L67-L68